Troubeshooting.md

```diff
@@ -0,0 +1,41 @@
+# Troubleshooting Guide
+
+## iOS Troubleshooting
```

Comment:
Trivial, but in the README we were listing alphabetically, Android then iOS.

Reply:
swapped